tests/nnfw_api/lib/GenModelTrain.h

```diff
@@ -116,10 +118,23 @@ class GenModelTrainContext : public GenModelTestContext
     _epoch = epoch;
   }
 
+  void enableTrainNodeUpdate(const int op_idx) { _trainable_ops_idx.emplace(op_idx); }
+
+  void disableTrainNodeUpdate(const int op_idx)
+  {
+    if (_trainable_ops_idx.find(op_idx) != std::end(_trainable_ops_idx))
+    {
+      _trainable_ops_idx.erase(op_idx);
+    }
+  }
+
+  const std::set<int> &getTrainableOpsIdx() const { return _trainable_ops_idx; }
+
 private:
   CircleBuffer _cpbuf;
   std::vector<TrainCaseData> _train_cases;
   int32_t _epoch;
+  std::set<int> _trainable_ops_idx;
```

Comment:
If we introduce `_trainable_ops_idx` in `nnfw_train_info`, we don't need to define it here.

Reply:
I see. I've applied the new logic to GenModelTrain staff ;)